Formula/docker-machine-driver-xhyve.rb

```diff
@@ -53,7 +53,8 @@ def caveats
   end
 
   test do
-    assert_match "xhyve-memory-size",
-    shell_output("#{Formula["docker-machine"].bin}/docker-machine create --driver xhyve -h")
+    assert_match "xhyve-memory-size", shell_output(
+      "#{Formula["docker-machine"].bin}/docker-machine create --driver xhyve -h",
```

Comment:
```suggestion
      "#{Formula["docker-machine"].bin}/docker-machine create --driver xhyve -h"
```

Reply:
@SMillerDev nope, this isn't valid with `rubocop`.